### ISSUE TITLE
EPro-PnP是否可以单独拿出来结合其他模型在KITTI数据集上训练？

### ISSUE THREAD

**Issue by MINGYUAN-W** [tjiiv-cprg/EPro-PnP#63]
您好！很感谢您能分享这么棒的工作的代码？请问EPro-PnP是否可以单独拿出来结合其他模型在KITTI数据集上训练？

**Comment by Lakonik** [comment]
Yes of course. The FCOS3D detector is a bit out of date now AFAIK, so I think there could be a lot more potential if you plug EPro-PnP into a more advanced model

**Comment by MINGYUAN-W** [comment]
Hello! I have another question, how long does it take for EPro-PnP-Det to train once on the nuScenes dataset?

**Comment by Lakonik** [comment]
About two days (for 12 epochs) using 2 RTX 3090 GPUs
